sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -565,6 +735,36 @@ export interface IssuerProperties {
   provider?: string;
 }
 
+/**
+ * The contact information for the vault certificates.
+ */
+export interface Contact {
+  /**
+   * Email address.
+   */
+  email?: string;
+  /**
+   * Name.
+   */
+  name?: string;
+  /**
+   * Phone number.
+   */
+  phone?: string;
+}
+
+export interface Contacts {
```

Comment:
This isn't a common model across languages. Is this idiomatic? Even so, below you use `Contact` and we haven't a class with just that name: either `AdministratorContact` or `CertificateContact`.

Reply:
This was extra it seems, so just removing it.